src/components/reports/smart-speaker-report.js

```diff
@@ -113,6 +109,18 @@ class SmartSpeakerReport extends React.Component {
 
     translateSortKey(key) {
         let sortKey = key;
+        
+        switch (key.toLowerCase()) {
+            case 'fullname':
+                sortKey = 'last_name';
+                break;
+            case 'emails':
+                sortKey = 'email_order';
```

Comment:
this seems not working
i do think that should be 'registration__email,member__email'